src/Form/Formbot.js

```diff
@@ -48,7 +52,7 @@ export default class Formbot extends React.Component {
   };
 
   get validatable() {
-    return Object.keys(this.props.validations);
+    return Object.keys(this.props.validationSchema || this.props.validations).length !== 0;
```

Comment:
If we want to get terse, we can rewrite this as:
`!!Object.keys(this.props.validationSchema || this.props.validations).length`

Reply:
Is this only used to map? Do we just want to return the keys?